src/modules/common-elements/constants.ts

```diff
@@ -603,3 +603,20 @@ export const NOTIFICATION_TYPES = {
   [CLAIM_REPORTING_FEES]: CLAIM_REPORTING_FEES,
   [UNSIGNED_ORDERS]: UNSIGNED_ORDERS
 };
+
+// Account View - Timeframe selection options 
+// # Timeframe readable names
+export const TIMEFRAMES = {
+  HOUR: "24 hr",
+  DAY: "1 Week",
+  WEEK: "1 Month",
+  ALL: "All Time",
+};
```

Comment:
I got confused a bit on constants names. DAY: “1 Week”

How about DAY: 24hr, ...

Reply:
yeah. that was a mixup